client/src/components/DataFiles/DataFilesBreadcrumbs/DataFilesBreadcrumbs.scss

```diff
@@ -1,10 +1,38 @@
 .breadcrumbs {
   /* ... */
   margin-right: 2em;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
```

Comment:
__Can Be More DRY__

Thank you for fixing this. Unless there is a technical limitation, please use the [`truncate-with-ellipsis` mixin](https://github.com/TACC/Core-Portal/commit/03468b336de77fc8dd24f866298b099d59a495e3) (so import and include rather than copy-pasting three rules).